Source/Initialization/WarpXInitData.cpp

```diff
@@ -542,12 +549,14 @@ void
 WarpX::AddExternalFields () {
     for (int lev = 0; lev <= finest_level; ++lev) {
         // FIXME: RZ multimode has more than one component for all these
-        if (m_p_ext_field_params->E_ext_grid_type == ExternalFieldType::read_from_file) {
+        if (m_p_ext_field_params->E_ext_grid_type == ExternalFieldType::read_from_file
+            || m_p_ext_field_params->E_ext_grid_type == ExternalFieldType::load_from_python) {
```

Comment:
I might be missing something here but it seems like this is a bug in `development`, no? The external field should always be added to the fine patch fields, even if it is constant or parsed. So shouldn't this just be:
```suggestion
        if (m_p_ext_field_params->E_ext_grid_type != ExternalFieldType::default_zero) {
```

Assuming that is correct, I don't think the `load_from_python` type is needed anymore.

Reply:
I believe the external fields are only allocated if the read_from_file, and now in this case the load_from_python flags are set for the external field grid type.

Reply:
This can be updated once #5104 is merged.

Reply:
This has been merged from PR #5104